Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php

```diff
@@ -108,13 +108,15 @@ class="ml-2"
                 <div class="form-group">
                     <label for="term_start_date" class="field-required">Term Start Date</label>
                     <input type="date" class="form-control" name="term_start_date" id="term_start_date" required="required"
-                        value="{{ now()->format('Y-m-d') }}">
+                           value="{{ \Carbon\Carbon::now()->subMonth()->startOfMonth()->toDateString() }}">
```

Comment:
Use `subMonthNoOverflow` function instead of subMonth as subMonth() function subtract 30 days from date, this  sometimes subtract 2 months instead of 1 month incase of February as it has 28 days only.  The subMonthNoOverflow() function takes care of it and only subtract 1 month.

```suggestion
                           value="{{ now()->subMonthNoOverflow()->startOfMonth()->toDateString() }}">
```